### PR TITLE
remove eventfeed keyboard shortcut

### LINKED ISSUE #3829: event feed keyboard shortcut is not accessible
## overview
the keyboard shortcut to reset the timescale on the event feed is not accessible because users cannot turn it off or remap it. to solve this, and because there is a button that does this already, let's remove the keyboard short cut. in the future, if we want to support keyboard shortcuts across automate it will need to be a full feature that allows folks to turn them off and remap them as needed.

## definition of done
- [ ] remove keyboard shortcut on event feed
- [ ] remove keyboard shortcut info from subheading on the event feed
- [ ] remove keyboard shortcut mentions from docs https://automate.chef.io/docs/event-feed/
- [ ] update all event feed screenshots, also work with docs, we prolly only need one on that page

### PR DESCRIPTION
**Pull request by susanev**
### :nut_and_bolt: Description: What code changed, and why?
we recently did an accessibility pass, and having any keyboard shortcuts without a way to turn them off or remap them causes us to fail. this removes the one keyboard short cut that existed in automate. if we want them in the future we will need to also build a way too turn them off and remap them.

### :chains: Related Resources
fixes #3829 

### :+1: Definition of Done
goodbye keyboard shortcut

### :athletic_shoe: How to Build and Test the Change
`rebuild automate-ui-devproxy`
`make serve` from `components/automate-chef-io`

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/automate-chef-io/content/docs/event-feed.md [hunk 1]
```diff
@@ -7,42 +7,34 @@ bref = ""
...
 Use the event feed for actionable insights and operational visibility.
-The guitar strings and improved query language help you drill into infrastructure & compliance automation events quickly to isolate errors.
+The guitar strings help you drill into infrastructure & compliance automation events quickly to isolate errors.
+
+The event feed shows the time of the event, its type, the object acted on, the action, and the initiating action.
 
 Users require permission for the `event:*` action to view and interact with *Event Feed*. Filter and search ability in *Event Feed* requires user permission for the `infra:nodes:list` action.
 
-## Event Guitar Strings
+![Event feed](/images/docs/event-feed.png)
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/event-feed.md:19]: @susanev Why is event-feed.png cited here when it's deleted in this PR?

Comment (susanev) [reply components/automate-chef-io/content/docs/event-feed.md:19]: its only 28% deleted, thats what it shows when you update the img

Comment (susanev) [reply components/automate-chef-io/content/docs/event-feed.md:19]: event-feed-change

Comment (mjingle) [reply components/automate-chef-io/content/docs/event-feed.md:19]: Ah ha! Okay. Now I get it. Thank you for confirming. Will have additional changes shortly

**Pull request merged**